independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/resolver/maven/options/BootstrapMavenOptions.java

```diff
@@ -90,7 +93,7 @@ public String getOptionValue(String name) {
 
     public String[] getOptionValues(String name) {
         final Object o = options.get(name);
-        return o == null ? null : (String[]) o;
+        return o == null ? null : o instanceof String ? new String[] { o.toString() } : (String[]) o;
```

Comment:
Just curious, was there a case when it actually was a `String`? Which option was that?

Reply:
I think it was these ones: https://github.com/quarkusio/quarkus/blob/8f0d1587609fd1d055caaa92bce99f70998b20b3/independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/resolver/maven/options/BootstrapMavenOptionsParser.java#L25

Reply:
If you recall how to reproduce that, please let me know. If not we'll merge it anyway. But I'd like to see the stacktrace.

Reply:
I can't actually remember, and it might not even be a problem any more as this PR has gone through so many changes.